### PR TITLE
Feature/301968 sheep which endemics package

### PR DESCRIPTION
**Pull request by JulieLev**

### CODE REVIEW

File: app/routes/endemics/sheep/ewe-tests.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const { urlPrefix } = require('../../../config')
+const { endemicsSheepEndemicsPackage, endemicsSheepEweTests } = require('../../../config/routes')
+
+const pageUrl = `${urlPrefix}/${endemicsSheepEweTests}`
+const backLink = endemicsSheepEndemicsPackage
```
Comment (alex742) [inline app/routes/endemics/sheep/ewe-tests.js:5]:
```suggestion
const backLink = `${urlPrefix}/${endemicsSheepEndemicsPackage}`
```

File: app/config/routes.js [hunk 2]
```diff
@@ -25,7 +25,8 @@ module.exports = {
   endemicsConfirmation: 'endemics/confirmation',
   endemicsVaccination: 'endemics/vaccination',
   endemicsDiseaseStatus: 'endemics/disease-status',
-  endemicsEndemicsPackage: 'endemics/endemics-package',
+  endemicsSheepEndemicsPackage: 'endemics/sheep-endemics-package',
+  endemicsSheepEweTests: 'endemics/sheep/ewe-tests',
```
[the remark this answers is not in the document]
Comment (JulieLev) [reply app/config/routes.js:29]: There are going to be more sheep test pages - one for each package selected (there are 6 options) so I thought they might be better organised as a group together - I could do it either way however. What do you think?

File: app/routes/endemics/sheep/ewe-tests.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+module.exports = {
+  method: 'GET',
+  path: pageUrl,
+  options: {
+    auth: false,
```
[the remark this answers is not in the document]
Comment (JulieLev) [reply app/routes/endemics/sheep/ewe-tests.js]: Done

File: app/config/routes.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (JulieLev) [reply app/config/routes.js]: All switched to endemics/sheep-tests

File: app/views/endemics/ewe-tests.njk [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        href: backLink
+      }) }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row govuk-width-container govuk-!-static-margin-top-0">
+    <div class="govuk-grid-column-two-thirds govuk-!-static-padding-0">
+      <h1 class="govuk-heading-l">What did the vet test or sample for?</h1>
+  </div>
+{% endblock %}
```
Comment (alex742) [inline app/views/endemics/ewe-tests.njk:18]: Should this file be renamed to sheep-tests to match what is in the routes?

Comment (JulieLev) [reply app/views/endemics/ewe-tests.njk]: It should... done!

**Review by alex742 — APPROVED** [review]

**Pull request merged**